build.gradle

```diff
@@ -140,10 +140,14 @@ task downloadAndInstallJRuby(dependsOn: verifyFile, type: Copy) {
 def jrubyBin = "${projectDir}/vendor/jruby/bin/jruby" +
   (System.getProperty("os.name").startsWith("Windows") ? '.bat' : '')
 
+def gemPath = "${projectDir}/vendor/bundle/jruby/2.3.0"
+
 def rakeBin = "${projectDir}/vendor/jruby/bin/rake"
 
 task installTestGems(dependsOn: downloadAndInstallJRuby, type: Exec) {
   workingDir projectDir
+  environment "GEM_PATH", gemPath
```

Comment:
@colinsurprenant this only helps you locally if you don't have `rvm` in your `bash` environment (sadly enough :(). That why I put the gem path adjustment into the `bash` script in https://github.com/elastic/logstash/pull/8896/files. The environment you set here is not making it's way into the actual execution because it's overwritten as soon as `vendor/jruby/bin/ruby` is loaded

Reply:
the environment should work regardless of if you use any Ruby version manager or not ...  
I am using rbenv, and this seems to work fine. Will review.